app/views/shared/_children-list.html.erb

```diff
@@ -1,7 +1,7 @@
 <ul class="govuk-list govuk-list--bullet">
-  <% browse_page.sorted_children.each do |child_tag| %>
-    <li>
-      <%= link_to(child_tag.title, polymorphic_path(child_tag), class: "govuk-link govuk-!-font-size-16") %>
+  <% page.sorted_children.each do |child_tag| %>
+    <li class="govuk-!-font-size-14 govuk-tag-s">
```

Comment:
I believe that `govuk-tag-s` is already applied on the tag - https://github.com/alphagov/collections-publisher/blob/main/app/helpers/status_helper.rb#L37